db/schema.rb

```diff
@@ -18,6 +18,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.string "password_digest"
+    t.index ["email"], name: "index_users_on_email", unique: true
```

Comment:
[Q] from where you get it ?

Reply:
in chapter 6, i doesn't run rails db:migrate. chapter 7, i run rails db:migrate:reset.